core/src/main/java/org/apache/iceberg/BaseIncrementalAppendScan.java

```diff
@@ -131,6 +126,17 @@ public CloseableIterable<CombinedScanTask> planTasks() {
     return TableScanUtil.planTasks(splitFiles, targetSplitSize(), splitLookback(), splitOpenFileCost());
   }
 
+  private long toSnapshotIdExclusive() {
```

Comment:
Isn't `toSnapshotId` always inclusive, not exclusive?

Reply:
good catch. I meant inclusive. somehow typed exclusive